scripts/build/TestImpactAnalysis/tiaf.py

```diff
@@ -310,6 +311,13 @@ def run(self, commit: str, src_branch: str, dst_branch: str, s3_bucket: str, s3_
         args.append(f"--suite={suite}")
         logger.info(f"Test suite is set to '{suite}'.")
 
+        # Exclude tests
+        if exclude_file is not None:
```

Comment:
nit- It's more Pythonic to write
`if exclude_file:`
which will be True if exclude_file is not 0, empty string, or None